php/context/class-fieldmanager-context-quickedit.php

```diff
@@ -1,66 +1,81 @@
 <?php
+/**
+ * Class file for Fieldmanager_Context_QuickEdit.
+ *
+ * @package Fieldmanager_Context_Storable
+ */
 
 /**
  * Use fieldmanager to add fields to the "quick edit" (post list inline editing)
  * and save data primarily to post meta.
- *
- * @package Fieldmanager_Context
  */
 class Fieldmanager_Context_QuickEdit extends Fieldmanager_Context_Storable {
 
 	/**
-	 * @var string
 	 * Title of QuickEdit box; also used for the column title unless $column_title is specified.
+	 *
+	 * @var string
 	 */
 	public $title = '';
 
 	/**
+	 * Override $title for the column in the list of posts.
+	 *
 	 * @var string
-	 * Override $title for the column in the list of posts
 	 */
 	public $column_title = '';
 
 	/**
-	 * @var callback
 	 * QuickEdit fields are tied to custom columns in the list of posts. This callback should return a value to
 	 * display in a custom column.
+	 *
+	 * @var callback
```

Comment:
We seem to use `@var callback` and `@var callable` interchangeably. Worth [etc]?